demo/to0scheduler/to0scheduler.env

```diff
@@ -1,94 +1,98 @@
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
+# Add proxy and other information here.
+# Proxy to be added as below for hostname 'company-proxy.com' and ports 80 and 443:
+# -Dhttp.proxyHost=company-proxy.com -Dhttp.proxyPort=80 -Dhttps.proxyHost=company-proxy.com -Dhttps.proxyPort=443
+_JAVA_OPTIONS=
```

Comment:
Won't this reset the _JAVA_OPTIONS value passed from docker-compose file.

Reply:
This property should now be passed on as a build argument to docker-compose, and, has been removed from the .env files.